parcels/examples/example_moving_eddies.py

```diff
@@ -161,6 +162,7 @@ def fieldsetfile(mesh):
 @pytest.mark.parametrize('mode', ['scipy', 'jit'])
 @pytest.mark.parametrize('mesh', ['flat', 'spherical'])
 def test_moving_eddies_file(mode, mesh):
+    gc.collect()
```

Comment:
Why exactly is this necessary? Does it have something to do with pytest?

Reply:
It's somehow related with pytest, but more generally with Python.

In our computeTimeChunk, we are closing our netcdf files as we open new ones. But we always stay with 3 open files. We don't want to close them at the end of pset.execute(), which does not make sense.
We only close them at field deletion (`see new func __del__()`).

While a variable in Python is supposed to leave within its scope (variable defined within a function does not exist elsewhere), but there is no guarantee the variable is really deleted until the end of the full Python program.
-> But we can afford such issue in our pytests, since we need a field from previous test to be deleted to recover access to the nc files which are created. This is where we use the garbage collector (`gc`) that triggers object deletion

Reply:
While this makes sense for moving_eddies and peninsula, I still don't know why we need it for the ofam benchmark (see my last commit 1584056524e54410659c8f9b9ab05794cf9a559e).

Since I added that we close the datasets when reseting computeTimeChunk, ofam produces an Time out-of-bound if I don't call the gc. 
This happens Only in pytest and only if I test the other example_* before.
I'm wondering why...